### PR TITLE
Tweaks the curator's access and clothing to improve his treasure hunting ability

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
rscadd: Curator's fedora has a pocket, like all the other fedoras.
tweak: Treasure hunter suits can now hold large internal tanks, to ease the exploration of lavaland.
tweak: The curator can now access the Auxillary Base and open doors on the mining station.
/:cl:

So I'm adjusting some things with the curator, since when I was playing these were the things I noticed that kind of suck, so I'm changing them to suck less.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
The curator and the detective are canonically buddies. Both drink and smoke heavily with a side of noir and adventure. This is canon.

**Comment by Repukan** [comment]
Is this only to the mining doors?

Because if they have access to mining lockers the buff would be too strong.

**Comment by ExcessiveUseOfCobblestone** [comment]
why didn't someone think to subtype the fedora variants to fedora reeee

**Comment by RandomMarine** [comment]
P O W E R   C R E E P

**Comment by Expletive** [comment]
@Repukan The mining lockers (Assuming you mean the ones with kinetic accelerators) are "/obj/structure/closet/secure_closet/miner" and require GLOB.access_mining, but I've only added GLOB.access_mining_station, which is for the doors to the mining station, in the event you take the shuttle there from the auxillary base. GLOB.access_construction is for getting into, building, and then launching the aux base.

I

**Comment by Repukan** [comment]
Not sure why they would get access to the area supposed to be for science to build (not even miners have access to it.)

But that no locker thing is reassuring.

**Comment by FantasticFwoosh** [comment]
I've said it once and ill say it again, curators are two steps away from being literal xeno-archeologists but this is a reassuring change.

**Comment by optimumtact** [comment]
@FantasticFwoosh such a change is inevitable :wink:

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -182,7 +182,7 @@
 	item_state = "curator"
 	blood_overlay_type = "coat"
 	body_parts_covered = CHEST|ARMS
-	allowed = list(/obj/item/weapon/tank/internals/emergency_oxygen, /obj/item/weapon/melee/curator_whip)
+	allowed = list(/obj/item/weapon/tank/internals/emergency_oxygen,/obj/item/weapon/tank/internals/plasmaman,/obj/item/weapon/tank/internals/oxygen, /obj/item/weapon/melee/curator_whip)
```
Comment (lzimann) [inline code/modules/clothing/suits/jobs.dm]: Just change this to `/obj/item/weapon/tank/internals` already.

**Comment by Expletive** [comment]
@lzimann Done

**Comment by Iamgoofball** [comment]
when i get around to it I'm giving him a spacepod so cementing him as an explorer is a good thing

**Comment by ExcessiveUseOfCobblestone** [comment]
Hivelords.... Why did it have to be hivelords?

**Pull request merged**